gnomad_qc/v3/annotations/generate_sv_hists.py

```diff
@@ -103,41 +100,39 @@ def get_sample_age(sv_list: hl.Table) -> hl.Table:
         ),
         release=sample_meta[sv_list.s].release,
     )
-    logger.info(
```

Comment:
might be helpful to run this in the job for future reference (since anyone could look up the runtime ticket and see these counts in the output)